### PR TITLE
Created migration for these tables:

### PR DESCRIPTION
**Pull request by RussellSk**
- employee
- permission
- role
- role_permission

### CODE REVIEW

File: sandbox-backend/src/main/resources/db/changelog/changes/20210326_create_table_employee.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+                  constraints:
+                    nullable: false
+              - column:
+                  name: EMP_LAST_NAME
+                  type: VARCHAR(255)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: EMP_CREATED_AT
+                  type: DATETIME
```
Comment (Clasker1) [inline sandbox-backend/src/main/resources/db/changelog/changes/20210326_create_table_employee.yaml:38]: defaultValue: now() will automatically set current timestamp for created record. Might be useful for created at column

File: sandbox-backend/src/main/resources/db/changelog/changes/20210326_create_table_permission.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: PMN_NAME
+                  type: VARCHAR(255)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: PMN_CREATED_AT
```
Comment (Clasker1) [inline sandbox-backend/src/main/resources/db/changelog/changes/20210326_create_table_permission.yaml:22]: Same, please add current timestamp for created at column

File: sandbox-backend/src/main/resources/db/changelog/changes/20210326_create_table_role.yaml [hunk 3]
```diff
@@ -0,0 +1,35 @@
+databaseChangeLog:
+  - changeSet:
+      id: "20210326_create_table_role"
```
Comment (Clasker1) [inline sandbox-backend/src/main/resources/db/changelog/changes/20210326_create_table_role.yaml:3]: Don't need quotes here

File: sandbox-backend/src/main/resources/db/changelog/changes/20210326_create_table_role.yaml [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+                  type: VARCHAR(255)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: RL_DESCRIPTION
+                  type: VARCHAR(255)
+                  constraints:
+                    nullable: true
+              - column:
+                  name: RL_CREATED_AT
```
Comment (Clasker1) [inline sandbox-backend/src/main/resources/db/changelog/changes/20210326_create_table_role.yaml:27]: Same comment about default timestamp

**Review by Clasker1 — CHANGES_REQUESTED** [review]

**Comment by RussellSk** [comment]
Fixed issues.
Added default value to *_created_at.
Remove quotes from id: field.

**Review by Clasker1 — APPROVED** [review]

**Pull request merged**